blueocean-dashboard/src/main/js/PipelineRoutes.jsx

```diff
@@ -127,6 +125,27 @@ function onLeaveCheckBackground() {
 
 const trends = AppConfig.isFeatureEnabled('trends');
 
+function getActions(extensionPoint, params, callback) {
```

Comment:
@stuartrowe 
Perhaps if you could start by providing comments (PR comments like this one, not code comments although those could be useful too) describing the changes you're making?  

You asserted in your description that no new tests are needed because you've moved the existing tabs to use the extension point.  Is in this block what you're talking about.

Reply:
@bitwiseman I've annotated the changes to the best of my understanding.